src/thor/timedep_forward.cc

```diff
@@ -14,7 +14,7 @@ namespace thor {
 constexpr uint32_t kInitialEdgeLabelCount = 500000;
 
 // Number of iterations to allow with no convergence to the destination
-constexpr uint32_t kMaxIterationsWithoutConvergence = 800000;
+constexpr uint32_t kMaxIterationsWithoutConvergence = 1800000;
```

Comment:
oof more than double :(   i guess the good news is that we rarely reach this limit except for some extreme cases?

Reply:
right, it happens not very often